app/assets/stylesheets/auk/override_bootstrap.scss

```diff
@@ -93,3 +93,12 @@ body {
   margin-right: auto;
   width: 50%;
 }
+
+/*
+ * Sigma instances
+ */
+#graph {
+  width: 400px;
```

Comment:
Let's change this a bit:

```
#graph {
  width: 100%;
  min-height: 400px;
  margin: auto;
  border: solid 2px black;
}
```

*Rationale*:
* Responsive width;
* sets a minimum height - 400px seems to work well.
* and adds a border so people know how to scroll.